app/navigation/router.js

```diff
@@ -72,7 +75,7 @@ class Router extends React.Component {
                     renderLeftComponent={renderLeftComponent}
                     renderTitleComponent={renderTitleComponent}
                     renderRightComponent={renderRightComponent}
-                    style={navigationProps.headerStyle}
+                    style={[{backgroundColor: this.props.theme.sidebarHeaderBg}, navigationProps.headerStyle]}
```

Comment:
if the `navigationProps.headerStyle` has a backgroundColor set the intent is that it takes preference over the theme background color?